uptime-kuma/templates/serviceaccount.yaml

```diff
@@ -4,7 +4,7 @@ kind: ServiceAccount
 metadata:
   name: {{ include "uptime-kuma.serviceAccountName" . }}
   labels:
-    {{- include "uptime-kuma.labels" . | nindent 4 }}
```

Comment:
while it is just for formatting, please keep the "trim whitespace/newlines" operator on the lavels